### PR TITLE
WiFi: Rely on libs module for wlan interface

### PR DESCRIPTION
**Pull request by tombettany**
To support variable network interfaces, don't hard-code the network
interface to use.

To be merged after release of v2.3.0.

cc @Ealdwulf @convolu @radujipa

### CODE REVIEW

File: etc/network/interfaces [hunk 1]
```diff
@@ -14,9 +14,17 @@ iface eth0 inet manual
  post-up /sbin/udhcpc -p /var/run/udhcpc-eth0.pid --script=/etc/udhcpc/kano.script -S -i eth0
  pre-down pkill -F /var/run/udhcpc-eth0.pid 2>/dev/null || /bin/true
 
+allow-hotplug eth1
+iface eth0 inet manual
+ post-up /sbin/udhcpc -p /var/run/udhcpc-eth0.pid --script=/etc/udhcpc/kano.script -S -i eth1
+ pre-down pkill -F /var/run/udhcpc-eth0.pid 2>/dev/null || /bin/true
```
Comment (convolu) [inline etc/network/interfaces]: Is this eth0 right here?

Comment (convolu) [inline etc/network/interfaces]: Oh I see, disregard the above message.

**Comment by convolu** [comment]
Looks ok to me

File: etc/network/interfaces (hunk 1 above)
Comment (skarbat) [inline etc/network/interfaces]: I think you were right @convolu , both eth lines should match the same interface?

**Comment by tombettany** [comment]
Merging for testing.

**Pull request merged**